### PR TITLE
"Thanks for submitting" Dialog

### PR DESCRIPTION
**Pull request by nachtm**
This PR adds a dialog that pops up when a user submits a form. 

Currently, the flow is:
 - Submit POST request to google cloud functions. When we get a status 200 response...
 - Open "thank you" dialog. When the dialog is closed by the user...
 - Navigate back to the main map page

I'm open to suggestions on this. It also looks pretty bare-bones at the moment -- maybe it'd be worth bringing Holly in on it, actually.

image

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -78,17 +82,12 @@ class Form extends Component {
...
 
   handleSubmit = () => {
-    const { history, handleDrawerState, fromDrawer } = this.props;
-    let report = this.state;
+    let {thanksOpen, ...report} = this.state;
     delete report['media'];
-
     return axios.post(addReportUrl, report)
       .then(response => {
         if (response.status === 200) {
```
Comment (sayas01) [inline src/components/Form.js]: Would we need to handle errors or the wait period ? For example, display circular progres until we receive the OK status or an error message on some network or other failures?

File: src/components/Form.js [hunk 2]
```diff
@@ -593,6 +602,16 @@ class Form extends Component {
             Submit
           </Button>
         </ValidatorForm>
+        <Dialog
+          open={thanksOpen}
+          onClose={this.handleClose}
+        >
+          <DialogContent>
+            <DialogContentText>
+              Thank you for submitting! Your submission should appear on the map within a week.
```
Comment (sayas01) [inline src/components/Form.js]: I like this this short and to the point message. Katie had added a similar message in her email yesterday to be displayed on form submission. It says "Thank you for your submission! Please note that the system will display your observation on the map after a period of one week". I am okay with either of these messages as the essence of both is the same.

File: src/components/Form.js [hunk 3]
```diff
@@ -593,6 +602,16 @@ class Form extends Component {
...
+            </DialogContentText>
+          </DialogContent>
+        </Dialog>
```
Comment (sayas01) [inline src/components/Form.js]: The dialogue box looks good to me. We could also send a screenshot of this to Holly to see if any more styling is needed.

File: src/components/Form.js (hunk 1 above)
Comment (nachtm) [reply src/components/Form.js]: Good point -- I've handled these now :)

File: src/components/Form.js (hunk 2 above)
Comment (nachtm) [reply src/components/Form.js]: Updated!

**Pull request merged**